medusa/providers/torrent/html/zooqle.py

```diff
@@ -142,7 +142,7 @@ def parse(self, data, mode):
                     torrent_size = cells[3].get_text().replace(',', '')
                     size = convert_size(torrent_size) or -1
 
-                    pubdate_raw = cells[4].get_text()
+                    pubdate_raw = cells[4].get_text().replace('yesterday', '24 hours')
```

Comment:
limetorrents already have this replace, why not create a dict so all providers may use it instead of adding duplicate replaces?